packages/dashboard/src/widgets/StockWidget.tsx

```diff
@@ -2,24 +2,31 @@ import React from 'react';
 import {
   Grid,
   StatsPanel,
-  useOmSupplyApi,
-  useQuery,
   useTranslation,
   Widget,
 } from '@openmsupply-client/common';
-import { getStockCountQueryFn } from '../api';
+import { useItemStats, useStockCounts } from '../api';
+
+const LOW_MOS_THRESHOLD = 3;
 
 export const StockWidget: React.FC = () => {
-  const { api } = useOmSupplyApi();
-  const t = useTranslation(['app', 'dashboard']);
-  const { data, isLoading } = useQuery(
-    ['stock', 'count'],
-    getStockCountQueryFn(api),
-    { retry: false }
-  );
+  const t = useTranslation(['dashboard']);
+  const { data: expiryData, isLoading: isExpiryLoading } = useStockCounts();
+  const { data: itemStatsData, isLoading: isItemStatsLoading } = useItemStats();
+
+  const lowStockItemsCount =
+    itemStatsData?.filter(
+      item =>
+        item.stats.availableStockOnHand > 0 &&
+        item.stats.availableMonthsOfStockOnHand < LOW_MOS_THRESHOLD
+    ).length || 0;
+
+  const noStockItemsCount =
+    itemStatsData?.filter(item => item.stats.availableStockOnHand === 0)
+      .length || 0;
```

Comment:
Could have another `useItemCounts` hook and put this in there.. though that's pretty excessive 😁

Reply:
I was thinking to have the hook do the work.. was hoping this might be temporary anyway - something that the back end could simply return 🤷